gammapy/data/observations.py

```diff
@@ -910,3 +915,58 @@ def check_psf(self):
         except Exception:
             yield self._record(level="warning", msg="Loading psf failed")
             return
+
+
+def create_observation_from_fermi_files(
+    path,
+    events_filename="ft1_00.fits",
```

Comment:
the defaults here are meaningless, no?

Reply:
I think it's the default names of the files created by fermipy